packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -8,7 +12,7 @@
 
 ## 2.0.8
 
-* Mark iOS arm64 simulators as unsupported.
+* Add additional marker drag events.
```

Comment:
This is a merge problem, please revert the old 2.0.8 message

Reply:
done